grouper/constants.py

```diff
@@ -28,6 +31,9 @@
     (PERMISSION_CREATE, "Ability to create permissions within Grouper."),
     (PERMISSION_GRANT, "Ability to grant a permission to a group."),
     (PERMISSION_AUDITOR, "Ability to own or manage groups with audited permissions."),
+    (PERMISSION_ADMIN, "Ability to manipulate any permission."),
+    (USER_ADMIN, "Ability to to disable/enable any User account and modify its attributes."),
+    (GROUP_ADMIN, "Ability to approve/deny/revoke membership to any group."),
```

Comment:
Might be worth keeping these separate and preserving the existing semantic that `SYSTEM_PERMISSIONS` aren't automagically granted during `sync_db`.

It 'feels' cleaner to have grouper-administrators only get the admin permissions but maybe you can talk about why you merged them?